components/dashboard/DashboardBlurb.js

```diff
@@ -0,0 +1,32 @@
+import React from 'react'
+import { formatDecimal, formatKm } from '../../lib/utils/format'
+import { compareAsc, getYear } from 'date-fns'
+import { head } from 'ramda'
+
+export default function Blurb ({
+  km_roads_add,
+  buildings_add,
+  poi_add,
+  km_waterways_add,
+  km_coastlines_add,
+  km_coastlines_mod,
+  country_list,
+  edit_times,
+  username
+}) {
+  let sentence = 'you have mapped'
+  if (username) {
+    sentence = `${username} has mapped`
+  }
+  const firstYearEdited = getYear(head(edit_times.map(t => t.day).sort(compareAsc)))
+
+  let countryWord = 'countries'
+  const countryLength = country_list.length
+  if (countryLength === 1) {
+    countryWord = 'country'
+  }
+
+  return <h2 className='header--small width--shortened list--block'>
+    Since <mark>{firstYearEdited}</mark>, <mark>{sentence}</mark> <mark>{formatKm(km_roads_add)}</mark> of roads, <mark>{formatDecimal(buildings_add)}</mark> buildings, <mark>{formatDecimal(poi_add)}</mark> Points of Interest, <mark>{formatKm(km_coastlines_add + km_coastlines_mod)}</mark> of coastlines, and <mark>{formatKm(km_waterways_add)}</mark> of waterways in <mark>{country_list.length}</mark> <mark>{countryWord}</mark>.
```

Comment:
@LanesGood @kamicut We should take out green highlight from "you have mapped" in the Dashboard blurb -- keeps highlight on the important info and is more consistent with Campaigns blurb

Campaign blurb:
<img width="939" alt="screen shot 2019-01-28 at 10 51 19 am" src="https://user-images.githubusercontent.com/6155420/51847968-be2fb500-22ea-11e9-98bf-d5076237467c.png">

Dashboard blurb:
<img width="904" alt="screen shot 2019-01-28 at 10 51 35 am" src="https://user-images.githubusercontent.com/6155420/51847994-c8ea4a00-22ea-11e9-8d53-0f9759f3c478.png">


```suggestion
    Since <mark>{firstYearEdited}</mark>, {sentence} <mark>{formatKm(km_roads_add)}</mark> of roads, <mark>{formatDecimal(buildings_add)}</mark> buildings, <mark>{formatDecimal(poi_add)}</mark> Points of Interest, <mark>{formatKm(km_coastlines_add + km_coastlines_mod)}</mark> of coastlines, and <mark>{formatKm(km_waterways_add)}</mark> of waterways in <mark>{country_list.length}</mark> <mark>{countryWord}</mark>.
```

Reply:
@LanesGood Oh okay, yeah to change that I think you would need to change how [sentence](https://github.com/developmentseed/scoreboard/pull/243/files/7fcd48ad0d5d6cafd2008d597a3cadea2838eb0b#diff-5f99008942e84cbfc846e00da79dc5b0R17) is written.

I think it would be fine not to highlight since you're on that user's page and since the username highlight will blend into the year highlight, but if you disagree, you would just need to change the above logic.

Reply:
@LanesGood @kamicut so which do you want to do?
Once this is resolved we can merge assuming the CircleCI test failure is a fluke